packages/liveblocks-node/src/client.ts

```diff
@@ -929,6 +931,70 @@ export class Liveblocks {
     }
     return (await res.json()) as Promise<CommentData>;
   }
+
+  /**
+   * Creates a comment.
+   *
+   * @param params.roomId The room ID to create the comment in.
+   * @param params.threadId The thread ID to create the comment in.
+   * @returns The created comment.
+   */
+  public async createComment(params: {
+    roomId: string;
+    threadId: string;
+    comment: {
+      userId: string;
```

Comment:
I am brainstorming other names for this particular variable. What do you think of `data` instead of `comment`? Would it be more consistent to our other names? I know we use the term `data` on our returned value quite a lot - Examples:

``` ts
public async getThreads(...): Promise<{ data: ThreadData[] }>()


public async getRooms(...): Promise<{
  nextPage: string | null;
  data: RoomInfo[];
}>


public async getActiveUsers(...): Promise<{ data: RoomUser[] }
```

Reply:
I've updated the implementation of `createComment` to the following:

https://github.com/liveblocks/liveblocks/blob/666e47aca2c4365331421f7a50effaf3f44cc790/packages/liveblocks-node/src/client.ts#L1020-L1053